tfhe/src/core_crypto/fft_impl/fft128/crypto/bootstrap.rs

```diff
@@ -358,6 +358,7 @@ where
             fft: Fft128View<'_>,
             stack: PodStack<'_>,
         ) {
+            #[allow(clippy::transmute_undefined_repr)]
```

Comment:
is there something we can do for the undefined repr ?

Reply:
Would probably need a refactor to treat u128 differently

Reply:
yeah no worries, we'll go from the allow list in lib.rs and remove the allows progressively for what makes sense